frontend/package.json

```diff
@@ -66,6 +66,7 @@
     "react-color": "^2.18.1",
     "react-copy-to-clipboard": "^5.0.2",
     "react-dom": "^16.8.4",
+    "react-dropzone": "^11.0.2",
```

Comment:
Not a big deal, but running `yarn` removed a dependency in the yarn.lock, which might be nice.